### ISSUE TITLE
TBI - refactor object clones (JSON.parse(JSON.stringify(...)))

### ISSUE THREAD

**Issue by heimwege** [SAP/open-ux-tools#2694]
### Description (include screenshots)
instead of `JSON.parse(JSON.stringify(...)))` we should rather use [structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm)

### Value
Using `JSON` has several limitations w.r.t circular objects, Map, Set, Date, RegEx

### Architecture Elaboration
Does it requires architecture elaboration? 🤷🏻 

### Notes
`structuredClone` might not be supported on older browser versions. See https://caniuse.com/?search=structuredClone

### Tasks
- [x] search and replace

**Comment by heimwege** [comment]
@Klaus-Keller you thoughts?

**Comment by Klaus-Keller** [comment]
@heimwege, I agree, most of the code runs on node and using `structuredClone()` shouldn't cause any issue. Let's switch.

**Comment by heimwege** [comment]
fixed in #2738
